R/rstudioapi_util.R

```diff
@@ -43,7 +43,8 @@ extract_document_ranges <- function(vsc_selections) {
 }
 
 to_content_lines <- function(contents, ranges) {
-    content_lines <- strsplit(contents, "\n")[[1]]
+    content_lines <- strsplit(contents, "\n|\r\n|\r$")[[1]]
```

Comment:
Does `\r?\n` work here?

Reply:
In my test case that resulted in a trailing `\r` on the last line.

Reply:
Could do `\r?\n|\r$` but I thought at that point the alternation is just clearer.

Reply:
Very interesting. Then I think your regex covers those cases.